editor/import/3d/resource_importer_scene.cpp

```diff
@@ -1959,6 +1959,13 @@ void ResourceImporterScene::get_import_options(const String &p_path, List<Import
 }
 
 void ResourceImporterScene::handle_compatibility_options(HashMap<StringName, Variant> &p_import_params) const {
+	print_line("Handle compatibility options");
+	if (p_import_params.has("meshes/force_disable_compression") && !p_import_params.has("meshes/compress")) {
+		print_line("Invert Force Disable Compression");
+		// Handle legacy option which had inverted behavior.
+		p_import_params["meshes/compress"] = !p_import_params["meshes/force_disable_compression"];
+	}
```

Comment:
This is the part that never runs, even if the `.import` file contains this legacy option (added with a text editor, or present in a project made before this PR). `.import` files store values even for default values, so this import option should logically always be present in 3D scenes imported before this PR.